integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/CatalogsPage.java

```diff
@@ -164,6 +173,55 @@ public void clickViewCatalogBtn(String name) {
     }
   }
 
+  public void sendPostJsonRequest(String targetURL, String jsonInputString) {
+    HttpURLConnection connection = null;
+    try {
+      URL url = new URL(targetURL);
+      connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("POST");
+      connection.setRequestProperty("Content-Type", "application/json;charset=UTF-8");
+      connection.setRequestProperty("Accept", "application/vnd.gravitino.v1+json");
+
+      connection.setUseCaches(false);
+      connection.setDoOutput(true);
+
+      try (OutputStream os = connection.getOutputStream()) {
+        byte[] input = jsonInputString.getBytes(StandardCharsets.UTF_8);
+        os.write(input, 0, input.length);
+      }
+      int responseCode = connection.getResponseCode();
+      LOG.info("POST Response Code :: " + responseCode);
+      if (responseCode == HttpURLConnection.HTTP_OK) {
+        LOG.info("POST request success.");
+      } else {
+        LOG.info("POST request failed.");
+      }
+    } catch (Exception e) {
+      LOG.error(e.getMessage(), e);
+    } finally {
+      if (connection != null) {
+        connection.disconnect();
+      }
+    }
+  }
+
+  public void createTable(
```

Comment:
If you need create table, Please use gravitino client to do.
refer to [AuditCatalogMysqlIT.java#130](https://github.com/datastrato/gravitino/blob/main/catalogs/catalog-jdbc-mysql/src/test/java/com/datastrato/gravitino/catalog/mysql/integration/test/AuditCatalogMysqlIT.java#L130)